config.go

```diff
@@ -16,47 +16,53 @@ var parsersByName = map[string]Parser{
 	"yellowfin": YellowfinLogParser,
 }
 
-type SourceConfig struct {
-	Name     string `json:"name"`
-	Filename string `json:"filename"`
-	Parser   string `json:"parser"`
+type ServiceRegistryConfig struct {
+	Services []struct {
+		Sources []struct {
+			Name     string `json:"name"`
+			Filename string `json:"filename"`
+			Parser   string `json:"parser"`
+		} `json:"sources"`
```

Comment:
This should not be called "sources" in `service-registry.json`. The word "sources" implies nothing about logs. I would just call it "logs".